src/lib.rs

```diff
@@ -274,6 +274,33 @@ impl Element {
         self.children.iter_mut().find(predicate)
     }
 
+    /// Transverse element using an xpath-like string: root/child/a
+    pub fn find(&self, path: &str) -> Option<&Element> {
+        Self::find_path(&path.split('/').collect::<Vec<&str>>(), &self)
+    }
+
+    pub fn find_text(&self, path: &str) -> &str {
```

Comment:
It would probably be better to add a variant to the error enum and have a `Result<_, Error>` to signal that the requested path does not exist. Perhaps `Option<&str>` to signal that there's no text in the tag.

`pub fn find_text(&self, path: &str) -> Result<Option<&str>, Error>` or similar, so that library users can deal with the errors in an appropriate way.